tools/github/print-runtime-release-issue.ts

```diff
@@ -76,10 +85,15 @@ ${commonTemplate}
 ## Post Release
 - [ ] Publish the docker runtime image (trigger the github action "Publish Docker runtime tanssi")
   - \`gh workflow run "Publish Runtime Draft" -r 'master' ` +
-      `-f from=runtime-${previousVersion} -f to=runtime-${newVersion}\`
+      `-f from=runtime-${previousVersion}-para -f to=runtime-${newVersion}-para\`
 - [ ] Publish the docker runtime image (trigger the github action "Publish Docker runtime containers")
   - \`gh workflow run "Publish Runtime Draft" -r 'master' ` +
       `-f from=runtime-${previousVersion}-templates -f to=runtime-${newVersion}-templates\`
+- [ ] Publish the docker runtime image tanssi-solochains (trigger the github action "Publish Docker runtime tanssi-solochain")
+  - \`gh workflow run "Publish Runtime Draft" -r 'master' ` +
+      `-f from=runtime-${previousVersion}-starlight -f to=runtime-${newVersion}-starlight\`
+- [ ] Run the action Upgrade typescript API with ${newVersion}, push and empty commit and merge
```

Comment:
Why do we need it?

Reply:
dapp

Reply:
Got it, so we publish the specific version of it.